assets/airports/ulli.json

```diff
@@ -0,0 +1,2695 @@
+{
+    "airac": 1909,
+    "radio": {
+        "twr": "Pulkovo Tower",
+        "app": "Peterburg Approach",
+        "dep": "Peterburg Radar"
+    },
+    "icao": "ULLI",
+    "iata": "LED",
+    "magnetic_north": 11,
+    "ctr_radius": 120,
+    "ctr_ceiling": 26500,
+    "initial_alt": 3500,
+    "position": ["N59d48.01m0", "E030d15.45m0", "79ft"],
+    "rangeRings": {
+        "enabled": true,
+        "center": [59.800278, 30.2625],
+        "radius_nm": 5.0
+    },
+    "has_terrain": true,
+    "wind": {
+        "angle": 230,
+        "speed": 6
+    },
+    "arrivalRunway": "28L",
+    "departureRunway": "28R",
+    "defaultMaps": ["Base Map"],
+    "airspace": [
```

Comment:
multiple airspace sections are not yet fully supported. I'd suggest keeping these under an invisible airspace key (`_airspace`) and adding the outer perimeter as the airspace. Take a look at EDDF for an example